### DISCUSSION TITLE
Docuseal docker

### DISCUSSION THREAD

**Discussion by jxs714** [docusealco/docuseal#374] [Q&A] [answered]
Hello guys,

I am using Imunify for Plesk and after scanning an old docuseal install, and even updating to a new docker install (which should be clean), Docuseal is coming up with 2 infected files. Can someone shed some light on why this is the case and how to fix it?

![download](https://github.com/user-attachments/assets/7fb84422-ea06-4d7f-b173-661905f2164a)

**Comment by omohokcoj** [comment, marked as answer]
@jxs714 DocuSeal docker image doesn't contain node_module directory also we're not using next.js - the error you showed on the screenshot must be from some other docker image but not docuseal

**Reply by jxs714** [reply]
Hi @omohokcoj 

Thanks for the quick response. My apologies for the confusion. You can delete this post completely so it doesn't interfere with misinformation. It seems I had documenso (a similar alternative) files in the folder where it found the virus it even though I was using a proxy pointing to the docker of docuseal
